pages/community.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import Head from 'next/head'
+
+import {
+  getLatestIssues,
+  getLatestTopics,
+  getLatestPosts
+} from '../src/utils/api'
+
+import Community from '../src/components/Community'
+import Page from '../src/components/Page'
+import Subscribe from '../src/components/Subscribe'
+
+import { META_BASE_TITLE } from '../src/consts'
+
+const HeadInjector = () => (
+  <Head>
+    <link
+      rel="stylesheet"
+      type="text/css"
+      charSet="UTF-8"
+      href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick-theme.min.css"
+    />
+    <title>Community | {META_BASE_TITLE}</title>
+  </Head>
+)
```

Comment:
nitpick: this could be a `head` and used below as `{head}` since it's static.

Don't feel obligated to change this if you feel like this is a better solution.

Reply:
I actually moved it inside the component on the other pages already, so probably I'll do it here too.

Reply:
I already moved it inside the component for other pages, so I probably will do it here too.